docs/source/contributing.rst

```diff
@@ -39,9 +39,8 @@ To install ``mirdata`` for development purposes:
 
 
 We recommend to install `pyenv <https://github.com/pyenv/pyenv#installation>`_ to manage your Python versions
-and install all ``mirdata`` requirements. You will want to install the latest versions of Python 3.6 and 3.7.
-Once ``pyenv`` and the Python versions are configured, install ``pytest``. Make sure you installed all the pytest
-plugins to automatically test your code successfully. Finally, run:
+and install all ``mirdata`` requirements. Once ``pyenv`` and the Python versions are configured, install ``pytest``.
```

Comment:
are we skipping the sentences "you will want to install the latests version of Python 3.7 and 3.8"?

Reply:
maybe we could generalize it to "you will want to install the latest versions of Python"... if we want it to be here of course.